liveclasses/views.py

```
@@ -0,0 +1,53 @@
+from django.shortcuts import render, get_object_or_404, redirect
+from django.contrib.auth.decorators import login_required
```

Ruff (0.8.2)

`django.contrib.auth.decorators.login_required` imported but unused

Remove unused import: `django.contrib.auth.decorators.login_required`

(F401)